### PR TITLE
Social media icons should link to latest entered username #2429

### PR DESCRIPTION
**Pull request by TamashaM**
fixes [#2429](https://github.com/publiclab/plots2/issues/2429)
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by jywarren** [comment]
This is looking great -- I do wonder if we could name it more generally -- like maybe `get_last_value_of_power_tag` instead? What do you think? 

Thanks!!!

**Comment by Gauravano** [comment]
hi @TamashaM , could you please change the method name as asked by @jywarren , all other changes are fine.Thanks !!!

**Comment by jywarren** [comment]
Hi, just checking if you've gotten stuck on this or if you needed some help? Thank you!!!

**Comment by TamashaM** [comment]
Sorry for the delay. I changed the function name as suggested.

**Comment by Gauravano** [comment]
No problem @TamashaM , and thanks for completing it

File: app/models/user.rb [hunk 1]
```diff
@@ -166,6 +166,12 @@ def get_value_of_power_tag(key)
     tvalue
   end
 
+  def get_last_value_of_power_tag(key)
+    tname = self.user_tags.where('value LIKE ?' , key + ':%')
+    tvalue = tname.last.name.partition(':').last
+    tvalue
+  end
+
```
Comment (Gauravano) [inline app/models/user.rb]: Awesome

**Comment by jywarren** [comment]
Awesome! Great work, everybody! 👍 🎉 🤝

**Pull request merged**